cookbooks/aws-parallelcluster-slurm/recipes/update_head_node.rb

```diff
@@ -175,9 +175,8 @@ def update_nodes_in_queue(strategy, queues)
 
 replace_or_add "Update compute console logging" do
   path "/etc/parallelcluster/slurm_plugin/parallelcluster_clustermgtd.conf"
-  pattern "compute_console_logging_enabled*"
-  line "compute_console_logging_enabled = #{node['cluster']['config'].dig(:Monitoring, :Logs, :CloudWatch, :Enabled) || node['cluster']['slurm_console_logging']['enabled']}"
-  replace_only true
+  pattern "compute_console_logging_enabled.*"
+  line "compute_console_logging_enabled = #{node['cluster']['cw_logging_enabled']}"
```

Comment:
Currently, `cw_logging_enabled` setting doesn't support update during cluster update, so I think we can remove the code here

Reply:
Removed.